BGL/include/CGAL/boost/graph/Euler_operations.h

```diff
@@ -737,9 +737,9 @@ add_face(const VertexRange& vr, Graph& g)
                       patch_start, patch_end;
   // cache for set_next and vertex' set_halfedge
   typedef std::pair<halfedge_descriptor, halfedge_descriptor> NextCacheEntry;
-  typedef std::vector<NextCacheEntry>    NextCache;
+  typedef boost::container::small_vector<NextCacheEntry,9>    NextCache;
   NextCache next_cache;
-  next_cache.reserve(3 * n);
+  //next_cache.reserve(3 * n);  Unfortunately small_vector has no reserve
```

Comment:
[`boost::small_vector`](https://www.boost.org/doc/libs/1_75_0/doc/html/boost/container/small_vector.html) inherits all the members from [`boost::vector`](https://www.boost.org/doc/libs/1_75_0/doc/html/boost/container/vector.html), including [`reserve(size_type)`](https://www.boost.org/doc/libs/1_75_0/doc/html/boost/container/vector.html#idm46216567859600-bb)